build/Dockerfile-setup

```diff
@@ -18,33 +20,26 @@ RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-ins
 ADD https://repo1.maven.org/maven2/sqlline/sqlline/1.12.0/sqlline-1.12.0-jar-with-dependencies.jar /sqlline/sqlline.jar
 ADD https://repo1.maven.org/maven2/com/oracle/ojdbc/ojdbc8/19.3.0.0/ojdbc8-19.3.0.0.jar /sqlline/ojdbc.jar
 
-# RUN curl https://packages.microsoft.com/keys/microsoft.asc | apt-key add - && \
-# RUN curl https://packages.microsoft.com/config/debian/9/prod.list > /etc/apt/sources.list.d/mssql-release.list && \
-# RUN apt-get update && \
-# ACCEPT_EULA=Y apt-get install -y msodbcsql mssql-tools
```

Comment:
Why did you uncomment these lines? 🤔

Reply:
Consistency w/ the other file. If they really shouldn't be there, then why were they there?